File: Roadmap/08 - CLASES/java/Qv1ko.java
class Qv1ko {

    public static void main(String[] args) {
        
        C1 example1 = new C1();
        System.out.println(example1.toString());
        example1.setC1String("Example 1");
        System.out.println(example1.getC1String());

        C2 example2 = new C2();
        System.out.println(example2.toString());
        example2.setC2String("Example 2");
        System.out.println(example2.getC2String());

    }

}

class C1 {

    String c1String;

    C1() {
        this.c1String = "example";
    }

    C1(String value) {
        this.c1String = value;
    }

    public String getC1String() {
        return c1String;
    }

    public void setC1String(String c1String) {
        this.c1String = c1String;
    }

    @Override
    public String toString() {
        return "Class 1 value: " + c1String;
    }
    
}

class C2 {

    String c2String;

    C2() {
        this.c2String = "example";
    }

    C2(String value) {
        this.c2String = value;
    }

    public String getC2String() {
        return c2String;
    }

    public void setC2String(String c2String) {
        this.c2String = c2String;
    }

    @Override
    public String toString() {
        return "Class 2 value: " + c2String;
    }
    
}

File: Roadmap/08 - CLASES/php/qv1ko.php
<?php

    class C1 {

        private $c1String;

        public function __construct($value = "example") {
            $this->c1String = $value;
        }

        public function getC1String() {
            return $this->c1String;
        }

        public function setC1String($c1String) {
            $this->c1String = $c1String;
        }

        public function __toString() {
            return "Class 1 value: " . $this->c1String;
        }

    }

    class C2 {

        private $c2String;

        public function __construct($value = "example") {
            $this->c2String = $value;
        }

        public function getC2String() {
            return $this->c2String;
        }

        public function setC2String($c2String) {
            $this->c2String = $c2String;
        }

        public function __toString() {
            return "Class 2 value: " . $this->c2String;
        }

    }

    function main() {

        $example1 = new C1();
        echo $example1 . "\n";
        $example1->setC1String("Example 1");
        echo $example1->getC1String() . "\n";

        $example2 = new C2();
        echo $example2 . "\n";
        $example2->setC2String("Example 2");
        echo $example2->getC2String() . "\n";

    }

    main();

?>
